src/components/Table/CustomTable.scss

```diff
@@ -0,0 +1,134 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap");
+
+@import '../../utilis/_mixins.scss';
+@import '../../utilis/_colors.scss';
+
+.table-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  .pagination-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    margin-top: 16px;
+
+    .pagination-info {
+      @include gilroy-medium();
+      color: #6C6D75;
+      display: flex;
+      align-items: center;
+    }
+    .ant-pagination-total-text {
+      display: none;
+    }
+  }
+}
+.show-total {
+  margin-left: auto;
+  padding-left: 20px;
+  @include gilroy-medium();
+  color: $gray-color;
+}
+
+.table {
+  border: 1px solid $border-color;
+
+  .ant-table-thead > tr > th {
+    @include gilroy-medium500;
+    color: $dark-gray-color;
+  }
+}
+
+.table tr {
+  border-bottom: 1px solid $border-color;
+  td {
+    @include gilroy-medium();
+    color: $gray-color;
+  }
+}
+
+.custom-pagination {
+  display: flex;
+  align-items: center;
+
+  .ant-pagination-prev,
```

Comment:
here you have example how to change antd arrows for prev and next page https://codesandbox.io/s/pagination-settings-antd-4-21-6-forked-0422vs?file=/demo.js